libraries.json

```diff
@@ -748,5 +748,10 @@
     "github": "Kodein-Framework/Kodein-DB",
     "category": "DataBase",
     "maven": "https://dl.bintray.com/kodein-framework/Kodein-DB/org/kodein/db/kodein-db/"
+  },
+  {
+    "github": "corbella83/PopKorn",
+    "category": "DI",
+    "maven": "https://repo1.maven.org/maven2/cc/popkorn/"
```

Comment:
please change address to `https://repo1.maven.org/maven2/cc/popkorn/popkorn/`

Reply:
Done!